package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "calculator",
+  "version": "1.0.0",
+  "repository": "https://github.com/next-step/js-calculator.git",
+  "author": "padawanR0k <padawanR0k@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "vitest -w",
+    "start": "live-server --port=5000 ./",
+    "coverage": "vitest run --coverage",
+    "test:e2e:open": "yarn cypress open",
+    "test:e2e:run": "yarn cypress run"
```

```suggestion
    "start": "live-server --port=5000 ./",
    "test": "vitest -w",
    "coverage": "vitest run --coverage",
    "test:e2e:open": "yarn cypress open",
    "test:e2e:run": "yarn cypress run"
```
이렇게 정렬시켜줘도 좋을 것 같아요!